ULC/Parser.hs

```diff
@@ -178,7 +178,7 @@ parseReplCommand s = case parse pCommand s of
 
 -- | Parse a command prefixed by a colon, or parse a raw term.
 pCommand :: Parser Command
-pCommand = pReds +++ pLet +++ pLoad +++ (T <$> pTerm) +++ pComment
+pCommand = spaces (pReds +++ pLet +++ pLoad +++ (T <$> pTerm) +++ pComment)
```

Comment:
why do you declare pReds but not `T <$> pTerm`?
For uniformity this would look better